packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -102,12 +104,18 @@ export const JETPACK_SCAN_PRODUCTS = <const>[
 	PRODUCT_JETPACK_SCAN_REALTIME_MONTHLY,
 ];
 
-// Social
+// Social Basic
 export const JETPACK_SOCIAL_PRODUCTS = <const>[
 	PRODUCT_JETPACK_SOCIAL_BASIC,
 	PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY,
 ];
 
+// Social Advanced
+export const JETPACK_SOCIAL_ADVANCED_PLANS = <const>[
+	PRODUCT_JETPACK_SOCIAL_ADVANCED,
+	PRODUCT_JETPACK_SOCIAL_ADVANCED_MONTHLY,
+];
+
```

Comment:
`JETPACK_SOCIAL_PRODUCTS` contains the Basic products and `JETPACK_SOCIAL_ADVANCED_PLANS` contains the Advanced products. It would probably be good to make these names consistent.

Reply:
Should we also change `JETPACK_SOCIAL_PRODUCTS` to `JETPACK_SOCIAL_BASIC_PRODUCTS`?

Reply:
Sure thing, it is being used in many files though, and the last time we tried tweaking that, we broke the pricing page. Due to that reason, I wonder if we can do that renaming in a different PR as a follow-up and test that separately, to reduce risk.